src/libs/Navigation/Navigation.ts

```diff
@@ -144,10 +144,6 @@ function getActiveRoute(): string {
         return '';
     }
 
-    if (currentRoute?.path) {
-        return currentRoute.path;
-    }
```

Comment:
Is the `currentRoute.path` incorrect here? Seems that the bug should be fixed in `navigationRef.current.getCurrentRoute()`

Reply:
Indeed `navigationRef.current.getCurrentRoute()` returns wrong path, it's connected to the fact that when choosing whole address (in addressLine1) we change values of the fields but we don't navigate, so the path is not changing. 
However the if condition was added to fix a [navigation issue](https://github.com/Expensify/App/issues/33001#issuecomment-1858822649) which no longer occurs (even without it)

Reply:
Ah I see. Although `getPathFromState` still suffers from the encoding issue it's seems okay to use for now.